config/cesm/config_grids.xml

```diff
@@ -969,28 +955,28 @@
 
     <!--  spectral element grids with 2x2 FVM physics grid -->
 
-    <model_grid alias="ne30pg2_ne30pg2_mg17" not_compset="_POP|_BLOM|_CLM">
+    <model_grid alias="ne30pg2_ne30pg2_mg17" compset="_DOCN|_SOCN">
```

Comment:
What about CLM?
```suggestion
    <model_grid alias="ne30pg2_ne30pg2_mg17" compset="_DOCN|_SOCN" not_compset="_CLM">
```

Reply:
These grids are fundamentally compatible with CLM - but it might be that there are not surface datasets yet created for this.